src/manifest.json

```diff
@@ -73,7 +75,391 @@
       ],
       "matches": [
         "https://www.google.com/*",
-        "http://www.google.com/*"
+        "http://www.google.com/*",
```

Comment:
Can we switch to `*://` for these domains to make this list more readable?

Reply:
["*://" is confusing as it includes ws(s) in some browsers but not others](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#Browser_compatibility) ... So I'd rather not.

Reply:
I guess it doesn't matter in this context? This isn't a permission to intercept requests, this is a content script injection permission, which only applies to frames. No such thing as a websocket document frame? But still, this is confusing ... I like the explicitness of http/https.